Source/Plugins/Core/com.equella.base/scalasrc/com/tle/beans/newentity/Entity.scala

```diff
@@ -27,6 +30,13 @@ class EntityID extends Serializable {
   @Column(length = 36)
   var uuid: String  = _
   var inst_id: Long = _
+
+  override def equals(obj: Any): Boolean = obj match {
+    case that: EntityID => that.uuid == that.uuid && this.inst_id == that.inst_id
+    case _              => false
+  }
+
+  override def hashCode(): Int = inst_id.hashCode + uuid.hashCode
```

Comment:
You'd probably get a more reliable hash with:

```suggestion
  override def hashCode(): Int = Objects.hash(inst_id, uuid)
```